src/components/table/table-cell/table-cell.style.js

```diff
@@ -25,7 +31,9 @@ const StyledTableCell = styled.td`
       }
       
       ${action && applyActionStyling(colors)}
-  `;
+
+      ${!isClassic(theme) && size ? applyModernPresentationStyling(size, isTextArea) : ''}
```

Comment:
Are we ok doing stuff like this? I'm totally fine with it, but I've had PRs rejected previously for including logic which refers to whether it's classic theme or not, outside of the actual classic style file - just checking so I know I can do this where it makes sense in future!

Reply:
There is only the single style file for TableCell so I think it's ok here, but I guess in scenarios where there are separate files it should be avoided. The style doesn't change when it's classic or modern as any overrides happen higher up based on the props received at that level.